src/Phinx/Console/Command/SeedCreate.php

```diff
@@ -173,10 +174,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // inject the class names appropriate to this seeder
         $contents = file_get_contents($this->getSeedTemplateFilename());
+
+        $config = $this->getConfig();
+        $namespace = $config instanceof NamespaceAwareInterface ? $config->getSeedNamespaceByPath($path) : null;
         $classes = array(
-            '$useClassName'  => 'Phinx\Seed\AbstractSeed',
-            '$className'     => $className,
-            '$baseClassName' => 'AbstractSeed',
+            '$defineNamespace' => null !== $namespace ? ('namespace ' . $namespace . ';') : '',
```

Comment:
Simply add ` '$namespace' => $namespace,`

Basically, I would need to see JUST the namespace in my template. This is independent to the `$defineNamespace` value.